src/helper.js

```diff
@@ -90,6 +91,10 @@ function getUserLogsPath(name, network) {
 	return path.join(this.HOME, "logs", name, network);
 }
 
+function getStoragePath() {
+	return path.join(this.HOME, "storage");
```

Comment:
Why not /var/lib/ instead? I have znc and sopel in there, with their service accounts' home directories set up in subfolders therein. Could be /var/lib/lounge